Packs/Silverfort/Integrations/Silverfort/Silverfort.py

```diff
@@ -0,0 +1,239 @@
+from CommonServerPython import *
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+# CONSTANTS
+API_KEY = demisto.params().get('apikey')
+UPDATE_REQ_RESPONSE = {'result': 'updated successfully!'}
+
+
+class Client(BaseClient):
+    def get_status_http_request(self):
+        """
+        initiates an http request to get the service status
+        """
+        params = {'apikey': API_KEY}
+        response = self._http_request(
+            method='GET',
+            url_suffix='getBootStatus',
+            params=params
+        )
+        return response
+
+    def get_upn_by_email_or_sam_account_http_request(self, domain, email=None, sam_account=None):
+        """
+        initiates an http request to get the upn by email or sam account
+        """
+        params = {'apikey': API_KEY, 'domain': domain}
```

Comment:
It can't be happen because get_upn function call get_upn_by_email_or_sam_account_http_request function with domain parameter. But, I changed the domain argument to be a default argument(default value is None)